test/appium/tests/test_general_verifications.py

```diff
@@ -24,6 +24,7 @@ def test_privacy_policy_is_accessible(self):
         signin_view = SignInView(self.driver)
         home_view = signin_view.create_user()
         profile = home_view.profile_button.click()
+        profile.about_button.scroll_to_element()
```

Comment:
@yevh-berdnyk 
@antdanchenko 
to avoid such cases when need to scroll to an element we can move scrolling to base button element level (under click() method).

see `base_element.py -> BaseButton -> click()`

```
def click(self):
    self.scroll_to_element().click()
    self.driver.info('Tap on %s' % self.name)
    return self.navigate()
``` 

This will allow us not to override click() method for every button which requires scrolling.
WDYT?

Reply:
I don't think that we need the scroll for each element since we can miss UX issues

Reply:
so, I am removing duplicated scroll in test and merging this PR if no objections.